config/src/config/network_config.rs

```diff
@@ -81,6 +82,50 @@ impl Default for NetworkConfig {
 }
 
 impl NetworkConfig {
+    /// Creates a validator network of with `nodes_in_network_count` validator nodes
+    pub fn default_validator_network_with_peers(
```

Comment:
looks like this method is only used in tests. Is it possible to define it there?